login.html

```diff
@@ -0,0 +1,132 @@
+                            <li>
+                                <a href="/login.html" class="icon-badge-container" title="account">
+                                    <i class="far fa-user text-md-size"></i>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="/product.html" class="icon-badge-container" title="shop">
+                                    <i class="far fa-store-alt text-md-size"></i>
+                                </a>
+                            </li>
+                            <li>
+                                <a href="/wishlist.html" class="icon-badge-container" title="whishlist">
+                                    <i class="far fa-heart text-md-size"></i>   
+                                </a>
+                            </li>
+                            <li>
+                                <a href="/cart.html" class="icon-badge-container" title="cart">
+                                    <i class="far fa-shopping-cart text-md-size"></i>   
+                                </a> 
+                            </li>
+                        </ul>
+                    </div>
+                </nav>   
+            </div>                                            
+        </header>
+
+        <main>
+            <section class="gutter-vh-xl">
+                <form class="form login-form" id="bb-login-form">
+                    <h4 class="text-center">LogIn</h4>
+                    <div>
+                        <input class="form-field" id="bb-login-email" type="email" name="email" placeholder="Email" required/>
+                    </div>
+                    <div>
+                        <input class="form-field" id="bb-login-password" type="password" name="password" placeholder="Password" minlength="8" required/>
+                    </div>
+                    <div class="flex-row justify-between align-center flex-wrap">
+                        <div>
+                            <input type="checkbox"/>
+                            <label>Remember Password</label>
+                        </div>
+                        <div>
+                            <a>Forgot Password</a>
+                        </div>
+                    </div>
+                    <button class="btn btn-primary" type="submit">Log In</button>
+                    <div class="form-group">
+                        <p class="text-center">Don't have an account? <a href="/signup.html">Sign up</a></p>
+                    </div>
+                </form>
+            </section>
+
+        </main>
+
+        <footer class="dark-bg padder-vh-xl light-text">
+            <div class="footer-grid">
+                <div class="footer-column footer-company-column">
+                    <div class="footer-logo">
+                        <div class="brand-logo">
+                            <img class="img-responsive" src="/images/bb-dark.png"/>
```

Comment:
Add ```alt``` attribute to image for accessibility purpose

Reply:
Added, Thanks